ui/src/app/views/resource-card/component.tsx

```diff
@@ -503,6 +558,13 @@ export class ResourceCard extends React.Component<ResourceCardProps, ResourceCar
                           workspaceNamespace={this.props.resourceCard.workspaceNamespace}
                           workspaceFirecloudName={this.props.resourceCard.workspaceFirecloudName}
                           closeFunction={() => this.setState({exportingDataSet: false})}/>}
+      {this.state.renamingDataSet &&
```

Comment:
Following above, could we do `this.state.renaming && this.isDataSet`? (Just to minimize state?)